templates/login.html

```diff
@@ -1,13 +1,26 @@
-{% load static %}
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <title>Login</title>
-    <link rel="stylesheet" href= {% static 'style.css' %}>
 </head>
 <body>
-This does nothing just allows you to get to the home page
-<a href="/home">Home</a>
+    <h2>Login</h2>
+    <form id="loginForm" action="/login" method="post">
+        <div>
```

Comment:
I'm not very good at HTML, but Rock mentioned how he looks for people using tables. Would it be possible to insert a table inside the form to represent username & password in their own rows? Or is what you've done the same thing as a table?

Reply:
I assume he meant for other things. I think this is simpler.